docs/troubleshooting.rst

```diff
@@ -193,3 +193,16 @@ Now build again::
     tutor images build
 
 All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+
+fatal: the remote end hung up unexpectedly / fatal: early EOF / fatal: index-pack failed
+----------------------------------------------------------------------------------------
+
+This issue can occur due to problems with your network while cloning edx-platform which is a fairly large repository.
```

Comment:
In what situation does this error occur? Unless I'm mistaken, this happens whenever we run `tutor images build ...` command, right? We should tell the reader that, such that they feel confident that the error that we are describing matches their experience.

Also, try to avoid the "you" pronoun whenever possible. For instance, some of the sentences can be rewritten as:

> This issue can occur due to network problems while cloning edx-platform which is a fairly large repository.

> First, try to run again the same command to see if it works.

Reply:
I've updated the title with the sample command.

I've removed the "you" pronoun as well.

Reply:
You left out the "your network" :) (and below as well)